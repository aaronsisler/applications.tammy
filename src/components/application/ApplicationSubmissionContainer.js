import React from 'react';
import { Link } from 'react-router-dom';

export default class ApplicationSubmissionContainer extends React.Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <div className="application_submission_container">
                <div className="application_submission_container_wrapper">
                    <div className="application_submission_container__content">
                        Thank you for your application! We will be in contact shortly.
                    </div>
                    <div className="application_submission_container__link">
                        <Link className="nav_link" to="/">Back to Positions</Link>
                    </div>
                </div>
            </div>
        );
    }
}
