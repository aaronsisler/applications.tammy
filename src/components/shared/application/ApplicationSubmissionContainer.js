import React from 'react';
import { connect } from 'react-redux';
// import PropTypes from 'prop-types';

export class ApplicationSubmissionContainer extends React.Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <div className="application_submission_container">
                application_submission_container
            </div>
        );
    }
}

const mapStateToProps = () => ({
})

const mapDispatchToProps = () => ({
})

export default connect(mapStateToProps, mapDispatchToProps)(ApplicationSubmissionContainer);

ApplicationSubmissionContainer.propTypes = {
};
