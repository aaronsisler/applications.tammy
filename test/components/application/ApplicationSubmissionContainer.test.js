import React from 'react';
import { shallow } from 'enzyme';
import ApplicationSubmissionContainer from 'Application/ApplicationSubmissionContainer';

describe('ApplicationSubmissionContainer', () => {
    let wrapper;

    const buildWrapper = () => {
        wrapper = shallow(
            <ApplicationSubmissionContainer />
        );
    };

    beforeEach(() => {
        buildWrapper();
    });

    it('should render ApplicationSubmissionContainer correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
