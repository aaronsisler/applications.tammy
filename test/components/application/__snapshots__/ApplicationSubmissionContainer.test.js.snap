// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationSubmissionContainer should render ApplicationSubmissionContainer correctly 1`] = `
<div
  className="application_submission_container"
>
  <div
    className="application_submission_container_wrapper"
  >
    <div
      className="application_submission_container__content"
    >
      Thank you for your application! We will be in contact shortly.
    </div>
    <div
      className="application_submission_container__link"
    >
      <Link
        className="nav_link"
        replace={false}
        to="/"
      >
        Back to Positions
      </Link>
    </div>
  </div>
</div>
`;
